### ISSUE TITLE
Start_Time and End_Time fields of 'Indicator's Valid_Time_Position cause validation error

### ISSUE THREAD

**Issue by howaryoo** [STIXProject/python-stix#306]
Hi,

using the code below to generate an Indicator in XML format:

```
from stix.core import STIXPackage
from stix.indicator import Indicator, ValidTime
from stix.utils import now

stix_package = STIXPackage()

indicator = Indicator(title="title")

valid_time = ValidTime(start_time=now())
indicator.add_valid_time_position(valid_time)

stix_package.add_indicator(indicator)

print stix_package.to_xml()

```

I get an invalid XML dateTime:

```
<indicator:Start_Time precision="second">2016-10-05 07:01:42.397913+00:00</indicator:Start_Time>
```
instead of the expected "2016-10-05**T**07:01:42.397913+00:00"

`stix-validator` fails with the error below: 

> $stix-validator.py ./sample_out.xml 
> [-] Performing xml schema validation on ./sample_out.xml
> ================================================================================
> [-] Results: ./sample_out.xml
> [!] XML Schema: False
>     [!] ./sample_out.xml:15:0:ERROR:SCHEMASV:SCHEMAV_CVC_DATATYPE_VALID_1_2_1: Element '{http://stix.mitre.org/Indicator-2}Start_Time': '2016-10-05 07:01:42.397913+00:00' is not a valid value of the atomic type 'xs:dateTime'.
> 
>

**Comment by gtback** [comment]
Bummer, [that](https://github.com/CybOXProject/mixbox/pull/33#issuecomment-252035451) didn't work. This isn't actually fixed until we update python-stix to require the new version of mixbox.

**Comment by gtback** [comment]
This should be fixed by 7f05e1a9f0631046fd6bfb015a8d21b7bdcba709
